src/mlagility/analysis/analysis.py

```diff
@@ -305,7 +314,10 @@ def explore_frame(
         if issubclass(type(local_var), torch.nn.Module):
             if type(local_var) in tracer_args.torch_activations:
                 return
-            model_type = build.ModelType.PYTORCH
+            if "dynamo_ctx" in local_var.__dict__:
```

Comment:
How durable is this? Is there a torch API for checking if a model is compiled?

Reply:
I couldn't find one... oh well, as long as your test provides coverage of this then we're good.

Reply:
Sounds good